src/Router.ts

```diff
@@ -201,13 +250,15 @@ export class Router<C extends Context> extends Evented {
 	private _history?: History;
 	private _routes: Route<Context, Parameters>[];
 	private _started?: boolean;
+	private _outletStack: Map<string, any> = new Map<string, OutletContext>();
+	private _outletRouteMap: Map<string, Route<any, any>> = new Map<string, Route<any, any>>();
```

Comment:
Another nit, but is this really a map of outlets to routes?  If it were, wouldn't it be a weak map of `Outlet`/`Route`?